src/rules/import-path-type.js

```diff
@@ -32,20 +32,21 @@ module.exports = {
 
         return {
             ImportDeclaration(node) {
-                let cwd = context.options[0].cwd || context.getCwd();
-                let fileName = path
+                const cwd = context.options[0].cwd || context.getCwd();
+                const fileName = path
                     .relative(cwd, context.getFilename())
                     .replace(/\\/g, '/');
 
                 const dirName = path.dirname(fileName);
 
                 // the base at which outside of this it should be absolute and inside it should be relative
-                const baseDir = dirName.split('/').slice(0, parts).join('/');
+                const baseDir =
+                    dirName.split('/').slice(0, parts).join('/') + '/';
 
                 const importPath = node.source.value;
-                const resolvedImportPath = importPath.startsWith('.')
-                    ? path.join(dirName, importPath).replace(/\\/g, '/')
-                    : importPath;
+                const resolvedImportPath = importPath.startsWith('.') ?

```

Comment:
Has some non-Prettier formatter done this, along with the changes to quote marks such that they now require escaping?

Reply:
I was formatting with prettier, but I haven't got round to setting up this project properly - so running lint-fix after prettier puts it back to requiring escaping...